### PR TITLE
add sorbet type checker to lsp

### PR DESCRIPTION
**Pull request by tvallois**
Hi,

The scope of this PR is to add default config for Sorbet, static type checker for Ruby (https://github.com/sorbet/sorbet).

image

image

### CODE REVIEW

File: LSP.sublime-settings [hunk 1]
```diff
@@ -537,6 +537,12 @@
       //   "diagnostics": true
       // },
     },
+    "sorbet": {
+      "command": ["srb", "tc", "--typed", "true", "--enable-all-experimental-lsp-features", "--lsp", "--disable-watchman"],
+      "languageId": "ruby",
+      "scopes": ["source.ruby"],
+      "syntaxes": ["Packages/Ruby/Ruby.sublime-syntax"]
```
[the remark this answers is not in the document]
Comment (tvallois) [reply LSP.sublime-settings]: No it's not intentional, i'll add it.

File: LSP.sublime-settings [hunk 2]
```diff
@@ -537,6 +537,16 @@
       //   "diagnostics": true
       // },
     },
+    "sorbet": {
+      "command": ["srb", "tc", "--typed", "true", "--enable-all-experimental-lsp-features", "--lsp", "--disable-watchman"],
+      "languageId": "ruby",
+      "scopes": ["source.ruby"],
```
Comment (rwols) [inline LSP.sublime-settings]: Could you also keep this in sync with solargraph's settngs? Solargraph includes `source.ruby.rails` here. After that, Looks mergeable to me.

Comment (tvallois) [reply LSP.sublime-settings]: is done

**Comment by predragnikolic** [comment]
Thanks :)

**Pull request merged**